src/sentry/api/endpoints/organization_stats_v2.py

```diff
@@ -131,7 +133,7 @@ class StatsApiResponse(TypedDict):
 
 @extend_schema(tags=["Organizations"])
 @region_silo_endpoint
-class OrganizationStatsEndpointV2(OrganizationEventsEndpointBase):
+class OrganizationStatsEndpointV2(OrganizationEndpoint):
```

Comment:
Now that handle_query_errors has been decoupled from the OrganizationEventsEndpointBase, we can extend the OrganizationEndpoint instead